### PR TITLE
Functions _ Ex 1 to 5

### PR DESCRIPTION
**Pull request by vazquezcabal**

### CODE REVIEW

File: stu12/functions.js [hunk 1]
```diff
@@ -5,35 +5,74 @@
...
+    printMessage("Helloo World!")
+    
 }
 
 const ex2 = () => {
-    console.log('Exercise 2')
+    const result = add(2, 3)
+    console.log(result) 
+
+    function add(arg1,arg2){
```
Comment (mburolla) [inline stu12/functions.js]: Let's convert all these functions to arrow functions.

**Review by vazquezcabal — COMMENTED** [review]
Changed to arrow functions

File: stu12/functions.js (hunk 1 above)
Comment (vazquezcabal) [reply stu12/functions.js]: Done

File: stu12/functions.js [hunk 2]
```diff
@@ -5,35 +5,75 @@
...
+
+    changeStuff(num, person1, person2)
+
+    console.log(num)
+    console.log(person1.firstName)
+    console.log(person2.firstName)
+
 }
 
-//
```
Comment (mburolla) [inline stu12/functions.js:27]: Can you move and group all your functions here?

Comment (vazquezcabal) [reply stu12/functions.js:27]: Done

**Pull request merged**